data/autoyast_sle12/autoyast_wicked_ppc64le.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <networking>
+    <keep_install_network config:type="boolean">true</keep_install_network>
+  </networking>
+  <bootloader>
+    <device_map config:type="list">
+      <device_map_entry>
+        <firmware>hd0</firmware>
+        <linux>/dev/vda</linux>
+      </device_map_entry>
+    </device_map>
+    <global>
+      <timeout config:type="integer">1</timeout>
+      <activate>true</activate>
+      <append>showopts plymouth.ignore-serial-consoles console=hvc0 console=tty mitigations=auto crashkernel=201M</append>
+      <boot_boot>false</boot_boot>
+      <xen_append>fadump= crashkernel=201M</xen_append>
+      <xen_kernel_append>vga=gfx-1024x768x16 vga=gfx-1024x768x16 crashkernel=201M\&lt;4G</xen_kernel_append>

```

Comment:
Not sure, I would prefer the `<![CDATA[` escaping more.
```suggestion
      <xen_kernel_append><![CDATA[vga=gfx-1024x768x16 vga=gfx-1024x768x16 crashkernel=201M<4G]]></xen_kernel_append>
```
I also don't see a explanation for `<` sign in crashkernel argument. But I only took a brief look. Can you give me a hint.

Reply:
> Not sure, I would prefer the `<![CDATA[` escaping more.
 ok will do this .  
> I also don't see a explanation for `<` sign in crashkernel argument. But I only took a brief look. Can you give me a hint.

no idea :) it is just copy/paste from "yast2 clone_system"  maybe even not needed but it will require extra testing cycles to answer this , decided not to invest into this...